x-pack/filebeat/module/zeek/dns/config/dns.yml

```
@@ -105,12 +110,54 @@ processors:
             evt.Put("event.duration", rttSec * 1000000000);
         }
 
+        function addTopLevelDomain(evt) {
+            var rd = evt.Get("dns.question.registered_domain");
+            if (rd == null) {
```

```suggestion
            if (!rd) {
```

This covers both null and undefined.